src/primitives/Sphere.js

```diff
@@ -1,18 +1,29 @@
 import * as THREE from 'three';
 
 import Mesh from './Mesh';
-import { DEFAULT_RADIAL_SEGMENTS } from '../utils/constants';
 
 class Sphere extends Mesh {
-  constructor(color, size = 1) {
-    super();
-    this.geometry = new THREE.SphereGeometry(
-      size,
-      DEFAULT_RADIAL_SEGMENTS,
-      DEFAULT_RADIAL_SEGMENTS
-    );
-    this.material = new THREE.MeshStandardMaterial();
-  }
+    constructor(color , size , widthSegments , heightSegments ) {
```

Comment:
Currently, the constructors in primitives take very limited arguments.
The radial segments is required by the three.js class but the viz using primitives don't need to send it.
That's the reason `Cylinder` and `Cone` class don't take the radial segments too. For consistency, at the moment I think we should remove them from the constructor with `DEFAULT_RADIAL_SEGMENTS` like other primitives.

Also `radius` might be more explicit than `size` since that could be confused with the diameter.